### ISSUE TITLE
Nerfacto settings for sparse image sets

### ISSUE THREAD

**Issue by briantcantwell** [nerfstudio-project/nerfstudio#887]
Hello!

I'm attempting to use a relative sparse set of images (48) with Nerfacto, but am not getting a valid reconstruction of my scene.  It works fine with Instant-NGP.  Are there settings to try perhaps?  Or is it more an issue of how Nerfacto works?

Thanks!

**Comment by tancik** [comment]
This shouldn't be a specific limitation of nerfacto. Can you provide some more info regarding how it is failing and what your input data looks like? When you say it works fine with instant-ngp, is this our implementation or the original implementation of NGP? Finally, how are you processing the data, are you using our scripts or another pipeline to get camera poses?

**Comment by briantcantwell** [comment]
I have a full orbit of cameras, but it's only 48 images total.   I can get a similar result with the test data (poster) if I set the "--train-split-percentage" parameter to .1 or so so it's only using a handful of cameras.   

In the case of instant-ngp, it's the original implementation.  I'm using the same transforms.json for both programs. When I look at it in the nerfstudio web viewer, the cameras appear to be in the correct spot.  

I'll attempt the nerfstudio instant-ngp implementation next.

**Comment by tancik** [comment]
Interesting, yea it really degrades in quality. There seems to be significant overfitting to the training views. I messed around with some configs, the following two seem to help with the poster scene, but definitely not as good as using all the images.

`--optimizers.proposal-networks.optimizer.lr 0.01 --pipeline.model.distortion-loss-mult 0`

**Comment by tancik** [comment]
Great to hear!
